File: retrofit/src/main/java/com/android/oleksandrpriadko/retrofit/HeaderInterceptorMashape.java
package com.android.oleksandrpriadko.retrofit;

import android.support.annotation.NonNull;

public class HeaderInterceptorMashape extends HeaderInterceptor {
    @SuppressWarnings("SpellCheckingInspection")
    private static final String MASHAPE_TITLE
        = "X-Mashape-Key";
    @SuppressWarnings("SpellCheckingInspection")
    private static final String MASHAPE_VALUE
        = "ZqZDokvSBlmshEMoSxzqhog0kOcOp1tnJgvjsnGOhZhcrGnywl";

    @NonNull
    @Override
    protected Header[] getHeaders() {
        Header header = new Header(MASHAPE_TITLE, MASHAPE_VALUE);
        return new Header[]{header};
    }
}

File: retrofit/src/main/java/com/android/oleksandrpriadko/retrofit/HeaderInterceptorMashape.kt
package com.android.oleksandrpriadko.retrofit

class HeaderInterceptorMashape : HeaderInterceptor() {

    override val headers: Array<HeaderInterceptor.Header>
        get() {
            val header = HeaderInterceptor.Header(MASHAPE_TITLE, MASHAPE_VALUE)
            return arrayOf(header)
        }

    companion object {
        private const val MASHAPE_TITLE = "X-Mashape-Key"
        private const val MASHAPE_VALUE = "ZqZDokvSBlmshEMoSxzqhog0kOcOp1tnJgvjsnGOhZhcrGnywl"
    }
}
